packages/client/wallets/aa/src/blockchain/wallets/FireblocksNCWallet.ts

```diff
@@ -174,3 +185,25 @@ const getDefaultAlgorithems = (): Set<TMPCAlgorithm> => {
     algorithms.add("MPC_CMP_ECDSA_SECP256K1");
     return algorithms;
 };
+
+function storeNCWDataToLocalStorage(walletId: string, deviceId: string) {
+    localStorage.setItem(NCW_DEVICE_ID_KEY, deviceId);
+    localStorage.setItem(NCW_WALLET_ID_KEY, walletId);
+}
+
+function getNCWDataFromLocalStorage(): { walletId: string; deviceId: string } | null {
+    const deviceId = localStorage.getItem(NCW_DEVICE_ID_KEY);
+    const walletId = localStorage.getItem(NCW_WALLET_ID_KEY);
+    if (deviceId && walletId) {
+        return {
+            walletId,
+            deviceId,
+        };
+    } else {
+        return null;
+    }
+}
+
+function storeAddressToLocalStorage(address: string) {
+    localStorage.setItem(NCW_ADDRESS_KEY, address);
+}
```

Comment:
hmm, this file is getting kinda long, and we can improve by hiding the details of `NCW_ADDRESS_KEY`, `NCW_WALLET_ID_KEY`, with an added layer of abstraction:
```
class LocalStorageRepository {
    private NCW_DEVICE_ID_KEY = "NCW-deviceId";
    private NCW_WALLET_ID_KEY = "NCW-walletId";
    private NCW_ADDRESS_KEY = "NCW-address";

    get ncwData() {
        const deviceId = localStorage.getItem(this.NCW_DEVICE_ID_KEY);
        const walletId = localStorage.getItem(this.NCW_WALLET_ID_KEY);
        return deviceId && walletId ? { walletId, deviceId } : null;
    }
    
    set ncwData(data: { walletId: string, deviceId: string } | null) {
        if (!data) {
            return 
        }
        const { walletId, deviceId } = data;
        localStorage.setItem(this.NCW_DEVICE_ID_KEY, deviceId);
        localStorage.setItem(this.NCW_WALLET_ID_KEY, walletId);
    }

    get ncwAddress() {
        return localStorage.getItem(this.NCW_ADDRESS_KEY)
    }

    set ncwAddress(address: string | null) {
        address ? localStorage.setItem(this.NCW_ADDRESS_KEY, address) : localStorage.removeItem(this.NCW_ADDRESS_KEY)
    }
}
```
and put this into its own file